ultrasonic/src/main/kotlin/org/moire/ultrasonic/service/OfflineMusicService.kt

```diff
@@ -257,9 +256,9 @@ class OfflineMusicService : MusicService, KoinComponent {
         if (children.isEmpty()) {
             return result
         }
-        val random = Random()
+        children.shuffle()
         for (i in 0 until size) {
-            val file = children[random.nextInt(children.size)]
+            val file = children[i % children.size]
```

Comment:
As far as I understand this logic,
if we request 40 random songs, but there are only 20 songs in total, 
it will return the 20 songs two times. 

Maybe we should change the 
`for (i in 0 until size)` so that the loop size is never larger than the amount of actual songs.

Reply:
Yeah it was the previous behavior and i put a comment FIXME about it but CI refused it so i removed it

I will update behavior to better match it

Reply:
You can use TODO, that is allowed by the CI :)